src/tab.js

```diff
@@ -130,7 +130,11 @@ SideTab.prototype = {
     this._setIcon("img/defaultFavicon.svg");
   },
   setSpinner() {
-    this._setIcon("img/loading-spinner.svg");
+    // Workaround until context-stroke is enabled in sidebar by default.
+    const isDarkTheme = document.body.classList.contains("dark-theme");
+    const isShrinked = document.getElementById("tablist").classList.contains("shrinked");
+    const frag = isDarkTheme && isShrinked && "#loading-invert" || "";
```

Comment:
Right, what I'm saying is this should be a background-image instead.
By default it would have the "globe" default favicon, then you could toggle a class named for example "show-spinner" that would do the right thing.